renovate.json

```diff
@@ -1,46 +1,80 @@
+
 {
- "extends": ["config:base"],
- "baseBranches": ["master"],
- "bumpVersion": null,
- "packageRules": [
-   {
-     "depTypeList": ["devDependencies"],
-     "updateTypes": ["minor"],
-     "automerge": false
-   }
- ],
- "prHourlyLimit": 5,
- "prConcurrentLimit": 5,
- "rangeStrategy": "bump",
- "renovateFork": true,
- "includeForks": true,
- "regexManagers": [
+  "extends":[
+    "config:base"
+  ],
+  "enabled": "true",
+  "baseBranches": ["master"],
+  "packageRules": [
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes/minikube",
-      "datasourceTemplate": "github-releases"
+      "depTypeList": ["devDependencies"],
+      "matchUpdateTypes": ["minor"],
+      "automerge": false
     },
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["KIND_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes-sigs/kind",
-      "datasourceTemplate": "github-releases"
+      "datasources": ["docker"],
+      "packageNames": ["kindest/node"],
+      "automerge": false,
+      "commitMessageTopic": "kindest/node"
+    },    
+    {
+      "datasources": ["pypi"],
+      "packageNames": ["ansible"],
+      "automerge": false 
     }
+
   ],
-  "assignees": ["fmuyassarov", "kashifest"],
-  "assigneesSampleSize": 1,
-  "reviewers": [
-      "furkatgofurov7",
-      "jan-est",
-      "lentzi90",
-      "macaptain",
-      "mboukhalfa",
-      "namnx228",
-      "smoshiur1237",
-      "Sunnatillo",
-      "wgslr",
-      "Xenwar"
-  ],
-  "reviewersSampleSize": 2
-}
+  "prHourlyLimit": 5,
+  "prConcurrentLimit": 5,
+  "rangeStrategy": "bump",
+  "renovateFork": true,
+  "includeForks": false,
+  "stabilityDays":"5",
+  "regexManagers": [
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KUBERNETES_VERSION:-\"(?<currentValue>.*?)\"}"],
```

Comment:
@Sunnatillo Have you managed to test this PR? Can Renovate bot detect if there is a newer K8s version available? I ask that question because I have a feeling that this can fail to detect the version which includes `v.` in it.

Reply:
yes. I have tested this on forked repository. Everything works fine. It is detecting new releases